javatests/arcs/core/analysis/ExpressionClaimDeducerTest.kt

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+package arcs.core.analysis
+
+import arcs.core.data.expression.PaxelParser
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class ExpressionClaimDeducerTest {
+
+  @Test
+  fun literal_number() {
+    val expr = PaxelParser.parse("5")
+
+    val actual = expr.accept(ExpressionClaimDeducer(), Unit)
+
+    assertThat(actual).isEqualTo(Deduction.Path())
+  }
+
+  @Test
+  fun literal_text() {
+    val expr = PaxelParser.parse("'hello'")
+
+    val actual = expr.accept(ExpressionClaimDeducer(), Unit)
+
+    assertThat(actual).isEqualTo(Deduction.Path())
+  }
+
+  @Test
+  fun literal_boolean() {
+    val expr = PaxelParser.parse("false")
+
+    val actual = expr.accept(ExpressionClaimDeducer(), Unit)
+
+    assertThat(actual).isEqualTo(Deduction.Path())
+  }
+
+  @Test
+  fun literal_null() {
+    val expr = PaxelParser.parse("null")
+
+    val actual = expr.accept(ExpressionClaimDeducer(), Unit)
+
+    assertThat(actual).isEqualTo(Deduction.Path())
+  }
+
+  @Test
+  fun single_variable() {
+    val expr = PaxelParser.parse("x")
```

Comment:
Can I have a test for `x + 1` please? The result should be `Deduction.Derive` instead of `Deduction.Equal`